release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -89,7 +89,7 @@ The {product-title} installation program for Google Cloud Platform (GCP) now cre
 [id="ocp-4-8-custom-console-routes-use-custom-domains-cluster-api"]
 ==== Custom console routes now use the new CustomDomains cluster API
 
-For `console` and `downloads` routes, custom routes functionally is now implemented to use the new `ingress` config route configuration API. The Console Operator already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent, because the route configuration via `console-operator` config is being deprecated.
+For `console` and `downloads` routes, custom routes functionality is now implemented to use the new `ingress` config route configuration API `spec.componentRoutes`. The Console Operator config already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent, because the route configuration via `console-operator` config is being deprecated.
```

Comment:
This LGTM! It looks like maybe this was previously peer reviewed, but a small rewording consideration that you can take or leave. :) 

> For `console` and `downloads` routes, custom routes functionality is now implemented to use the new `ingress` config route configuration API `spec.componentRoutes`. The Console Operator config already contained custom route customization, but for the `console` route only. The route configuration via `console-operator` config is being deprecated. Therefore, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration now takes precedent.